.github/workflows/check-update-pelorus-operator.yml

```diff
@@ -0,0 +1,31 @@
+name: Check for updates for pelorus operator
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  schedule:
+    - cron:  '0 0 * * 1' # Every Monday at 00:00 UTC
```

Comment:
I think weekly is enough, what you think?
I just entered a random date, we can choose when it is best to run it